docs/Developers/Review.rst

```diff
@@ -3,19 +3,51 @@ Reviewing
 
 This document is a guide to coala's review process.
 
+Am I Good Enough to Do Code Review?
+-----------------------------------
+
+**Yes,** if you already fixed a newcomer issue.
+
+But be careful: **bad reviews will scare away other contributors.**
```

Comment:
Since the first and following paragraph seem to foremost encourage reviewing, maybe remove this here and add it further down where you talk about being transparent and polite (l. 26).